settings.json.docker

```diff
@@ -587,5 +587,13 @@
   "loglevel": "${LOGLEVEL:INFO}",
 
   /* Override any strings found in locale directories */
-  "customLocaleStrings": {}
+  "customLocaleStrings": {},
+
+  "ep_delete_after_delay": {
```

Comment:
@tim-krieger Can you please check this configuration for the etherpad plugin `ep_delete_after_delay`. The logs contain error messages without this config.

Reply:
> We need to look into this error message and check how to fix this
> 
> ```
> etherpad-lite-app_prod-1       | [2022-08-21 21:25:05.161] [ERROR] console - DOUBLE SETTLE BUG IN HOOK FUNCTION (plugin: ep_push2delete, function name: /opt/etherpad-lite/node_modules/ep_push2delete/index:handleMessage, hook: handleMessage): Attempt to resolve via returned value but it already resolved via callback. Ignoring this attempt to resolve.
> ```
> 
> @tim-krieger Do you have an idea how to fix this?

This error is also mentioned in https://github.com/marcelklehr/ep_push2delete/issues/11 . Unfortunately, there is no fix for this.

@tim-krieger You should have the same error on  your current instance, is this right?

Reply:
Yes, we have the same error in our instance.